bucket/zaproxy.json

```diff
@@ -1,14 +1,14 @@
 {
-    "version": "2.11.1",
+    "version": "2.12.0",
     "description": "Integrated penetration testing tool for finding vulnerabilities in web applications",
     "homepage": "https://www.zaproxy.org",
     "license": "Apache-2.0",
     "suggest": {
-        "JRE": "java/adopt8-hotspot-jre"
+        "JRE": "java/adopt12-hotspot-jre"
```

Comment:
`java/adopt12-hotspot-jre` no longer exists.

`java/temurin-jre` and `java/temurin11-jre` is good, according to the release log and previous suggest.

> This is a bug fix and enhancement release, which now requires a minimum of Java 11.
> 
> [release log](https://www.zaproxy.org/docs/desktop/releases/2.12.0/)

Reply:
btw, the `autoupdate` and `checkver` for the app are correct, even no PR for version update it will be updated by Excavator.

Reply:
I believe we should use `java/temurin11-jre`